### DISCUSSION TITLE
Conditional stages

### DISCUSSION THREAD

**Discussion by lordscarlet** [nathanwoulfe/Plumber-2#77] [Q&A]
@nathanwoulfe, can you tell me about _Conditional stages_? I don't see them mentioned anywhere, and they don't seem to be working properly for me. Or there's more configuration that is not clear.

![image](https://user-images.githubusercontent.com/17951/111147592-08bf2a80-8561-11eb-8e46-a91774733c2a.png)

**Comment by lordscarlet** [comment]
This may actually be a bug.. After I went back to edit this Document-type approval flow, options were available.

![image](https://user-images.githubusercontent.com/17951/111150887-0a8aed00-8565-11eb-857f-a0f3425d4098.png)

**Comment by nathanwoulfe** [comment]
@lordscarlet that may well be a bug - it should show the second screenshot when the overlay is first opened. Will poke around and see what's up...

Conditional stages allow including particular workflow steps only when particular fields have changes. Use case could be, for example, only requiring HR to approve changes to staff profiles if the bodyText has changed.

It's would probably be more useful if it managed the entire workflow rather than stages - only require workflow approval when field x has changes... Could do both, I guess.

**Comment by nathanwoulfe** [comment]
Tonight's nightly fixes this (he says with fingers crossed). On my second point, that's effectively possible now by setting conditions for all stages of a workflow on the same field (if no one needs to approve the change, the workflow will complete immediately).
